{"cve_repaire_en": "", "cve_code_exp_url": "", "cve_id": "CVE-2017-15126", "cve_categories": ["category1", "category2"], "cve_code_detect_url": "", "cve_name": "cve_name", "cve_risk_level": "determine", "cve_detail_en": "A use-after-free flaw was found in fs/userfaultfd.c in the Linux kernel before 4.13.6. The issue is related to the handling of fork failure when dealing with event messages. Failure to fork correctly can lead to a situation where a fork event will be removed from an already freed list of events with userfaultfd_ctx_put().", "cve_name_en": "cve_name_en", "edit_time": "Sat Oct 12 15:52:52 2019", "cve_code_des_url": "", "cve_other_info": "", "modify_time": "2018-05-06", "cve_detail": "", "references": [{"URL": "http://www.securityfocus.com/bid/102516", "Description": "BID:102516"}, {"URL": "http://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=384632e67e0829deb8015ee6ad916b180049d252", "Description": "MISC:http://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=384632e67e0829deb8015ee6ad916b180049d252"}, {"URL": "https://access.redhat.com/security/cve/CVE-2017-15126", "Description": "MISC:https://access.redhat.com/security/cve/CVE-2017-15126"}, {"URL": "https://bugzilla.redhat.com/show_bug.cgi?id=1523481", "Description": "MISC:https://bugzilla.redhat.com/show_bug.cgi?id=1523481"}, {"URL": "https://github.com/torvalds/linux/commit/384632e67e0829deb8015ee6ad916b180049d252", "Description": "MISC:https://github.com/torvalds/linux/commit/384632e67e0829deb8015ee6ad916b180049d252"}, {"URL": "https://www.kernel.org/pub/linux/kernel/v4.x/ChangeLog-4.13.6", "Description": "MISC:https://www.kernel.org/pub/linux/kernel/v4.x/ChangeLog-4.13.6"}, {"URL": "https://access.redhat.com/errata/RHSA-2018:0676", "Description": "REDHAT:RHSA-2018:0676"}, {"URL": "https://access.redhat.com/errata/RHSA-2018:1062", "Description": "REDHAT:RHSA-2018:1062"}], "cve_repaire": "", "editor": "warcup", "add_time": "2018-01-14"}